config/initializers/content_security_policy.rb

```diff
@@ -46,3 +46,8 @@ def host_to_url(str)
 # For further information see the following documentation:
 # https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy-Report-Only
 # Rails.application.config.content_security_policy_report_only = true
+
+PgHero::HomeController.content_security_policy do |p|
```

Comment:
Ah, I could've done this the entire time instead of submitting a new feature to PgHero... :facepalm: